### DISCUSSION TITLE
Trail Renderer?

### DISCUSSION THREAD

**Discussion by Mechano-64** [yasirkula/UnityBezierSolution#31] [Ideas]
Hi, Love this package, really complete. However, I was wondering if a trail renderer based on splines could be made and bundled with the package. It would be really helpful since Unity's trail renderer is very jaggy and unreliable.

Keep up the good work

**Comment by yasirkula** [comment]
There is BezierLineRenderer component for LineRenderers; I can't visualize how you're planning to use TrailRenderer with spline. TrailRenderer adds new points to its trail as it moves around, in what aspect are you planning to integrate that component with a spline?

**Comment by Mechano-64** [comment]
I thought it would be useful for things like attacks, usually artists use particles to make trails for sword trails and such and while that works for simple swings, when a swing draws a complex shape then making and editing the trail effect becomes cumbersome and tedious. This is where a spline based trail renderer could come handy and provide better and more appealing results than the built in trail renderer

**Reply by yasirkula** [reply]
I see. I'll add a label to this Discussion and in my spare time, give it a try.
